test/test_eds.py

```diff
@@ -222,6 +222,7 @@ def test_export_eds_unknown_doctype(self):
         import io
         filelike_object = io.StringIO()
         self.addCleanup(filelike_object.close)
+        self.addCleanup(os.unlink, "filename")
```

Comment:
Note: this will fail if the first iteration of the subsequent test loop fails; no file named `filename` is created, thus `FileNotFoundError` is raised. This exposes another bug of `export_od`, namely that it creates the file on disk before actually checking if a valid format was provided.

Reply:
🎉  Isn't that why test cases exists? To find bugs I mean?

Reply:
I created #485 for tracking this.